### PR TITLE
Adds 8.7 to conf.yaml

### PR DESCRIPTION
**Pull request by KOTungseth**

### CODE REVIEW

File: conf.yaml [hunk 1]
```diff
@@ -1058,7 +1058,7 @@ contents:
               - title:      Enterprise Search Node.js client
                 prefix:     enterprise-search-node
                 current:    *stackcurrent
-                branches:   [ {main: master}, 8.6]
+                branches:   [ {main: master}, 8.7]
```
Comment (lcawl) [inline conf.yaml:1061]: >                branches:   [ {main: master}, 8.7]

Is this correct (i.e. removal of 8.6 instead of just appending 8.6)?

**Review by lcawl — APPROVED** [review]
One Q otherwise LGTM

File: conf.yaml (hunk 1 above)
Comment (KOTungseth) [reply conf.yaml]: Nope! Fixed.

**Comment by KOTungseth** [comment]
@elasticmachine run elasticsearch-ci/docs

**Comment by KOTungseth** [comment]
@elasticmachine run elasticsearch-ci/docs

**Comment by bmorelli25** [comment]
Don't we also need to add a `shared/versions/stack/8.7.asciidoc`?

**Comment by bmorelli25** [comment]
> Don't we also need to add a `shared/versions/stack/8.7.asciidoc`?

lol. I now see you've already done that :) https://github.com/elastic/docs/pull/2616

**Comment by KOTungseth** [comment]
@elasticmachine run elasticsearch-ci/docs

**Comment by KOTungseth** [comment]
@elasticmachine run elasticsearch-ci/docs

**Pull request merged**